tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -95,3 +99,49 @@ After(function (testCase) {
   console.log('\n  Result: ' + testCase.result.status + '\n')
   createdFiles.forEach(fileName => fs.unlinkSync(fileName))
 })
+
+Before(async function () {
+  const resp = await occHelper.runOcc(
+    [
+      'config:list'
+    ])
+  let stdOut = _.get(resp, 'ocs.data.stdOut')
+  if (stdOut === undefined) {
+    throw new Error('stdOut notFound, Found:', resp)
+  }
+  stdOut = JSON.parse(stdOut)
+  appConfigSettings = _.get(stdOut, 'apps')
+  coreAppConfigSettings = _.get(appConfigSettings, 'core')
+  if (appConfigSettings === undefined) {
+    throw new Error("'apps' was not found inside stdOut of response.")
+  }
+})
+
+After(async function () {
+  const afterConfigSetting = await occHelper.runOcc(
+    [
+      'config:list'
+    ])
+  let afterStdOut = _.get(afterConfigSetting, 'ocs.data.stdOut')
+  if (afterStdOut === undefined) {
+    throw new Error('stdOut notFound, Found:', afterStdOut)
+  }
+  afterStdOut = JSON.parse(afterStdOut)
+  const afterAppConfigSettings = _.get(afterStdOut, 'apps')
+  const afterCoreAppConfigSettings = _.get(afterAppConfigSettings, 'core')
+  if (afterAppConfigSettings === undefined) {
+    throw new Error("'apps' was not found inside stdOut of response.")
+  }
+  for (const index in coreAppConfigSettings) {
```

Comment:
I think it would be better to do a diff of the objects which would simplify code here (but, maybe, I'm overthinking?).

What do you think?